doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -68,30 +94,159 @@ between tenants
+while providing extra telemetry information on transactions between services.
+
+Kubernetes expects that all API communication in the cluster is encrypted by
+default with TLS, and the majority of installation methods will allow the
+necessary certificates to be created and distributed to the cluster components.
+Note that some components and installation methods may enable local ports over
+HTTP and administrators should familiarize themselves with the settings of each
+component to identify potentially unsecured traffic.
+
+### 5.4.4 API Authentication, API Authorization
 Secure all connections to a Kubernetes Cluster. Adopt the following security
 authentication mechanisms:
- - Configure user roles and access levels to provide segregation of duties (RBAC)
- - Use multi-factor authentication for all administrative access
- - Use token-based or certificate-based service and session authentication mechanisms
- - Integrated with existing identity management platforms e.g SAML, AD, etc. for
- access control
+* Configure user roles and access levels to provide segregation of duties (RBAC)
+* Use multi-factor authentication for all administrative access
+* Use token-based or certificate-based service and session authentication
+  mechanisms
+* Integrated with existing identity management platforms e.g SAML, AD, etc. for
+  access control
+
+### 5.4.5 Restrict access to etcd and encrypt contents within etcd
+etcd is a critical Kubernetes component which stores information on state and
+secrets, and it should be protected differently from the rest of your cluster.
+Write access to the API server's etcd is equivalent to gaining root on the
+entire cluster, and even read access can be used to escalate privileges fairly
+easily.
+
+The Kubernetes scheduler will search etcd for pod definitions that do not have a
+node. It then sends the pods it finds to an available kubelet for scheduling.
+Validation for submitted pods is performed by the API server before it writes
+them to etcd, so malicious users writing directly to etcd can bypass many
+security mechanisms - e.g. PodSecurityPolicies.
+
+Administrators should always use strong credentials from the API servers to
+their etcd server, such as mutual auth via TLS client certificates, and it is
+often recommended to isolate the etcd servers behind a firewall that only the
+API servers may access.
+    
+### 5.4.6 Controlling access to the Kubelet
+Kubelets expose HTTPS endpoints which grant powerful control over the node and
+containers. By default Kubelets allow unauthenticated access to this API.
+Production clusters should enable Kubelet authentication and authorization
+    
+### 5.4.7 Securing Kubernetes Dashboard 
+The Kubernetes dashboard is a webapp for managing your cluster. It it is not a
+part of the Kubernetes cluster itself, it has to be installed by the owners of
+the cluster. Thus, there are a lot of tutorials on how to do this.
```

Comment:
Why "thus"

Reply:
Since it is not a part of the standard Kubernetes installation, it's not covered as part of the official docs. The community has created a lot of materials, each in it's own way and the intent is to highlight all methods may not be secure

Reply:
```suggestion
the cluster; a number of tutorials show how to do this.
```